src/app/view/perspectives/kanban/kanban-perspective.module.ts

```diff
@@ -27,43 +27,53 @@ import {SharedModule} from '../../../shared/shared.module';
 import {KanbanPerspectiveRoutingModule} from './kanban-perspective-routing.module';
 import {KanbanConfigComponent} from './config/kanban-config.component';
 import {KanbanColumnsComponent} from './columns/kanban-columns.component';
-import {KanbanCollectionConfigComponent} from './config/collection/kanban-collection-config.component';
+import {KanbanStemConfigComponent} from './config/stem/kanban-stem-config.component';
 import {KanbanColumnComponent} from './columns/column/kanban-column.component';
 import {KanbanColumnHeaderComponent} from './columns/column/header/kanban-column-header.component';
 import {KanbanColumnFooterComponent} from './columns/column/footer/kanban-column-footer.component';
-import {KanbanColumnDocumentsPipe} from './pipes/kanban-column-documents.pipe';
+import {KanbanColumnCardsPipe} from './pipes/kanban-column-cards.pipe';
 import {KanbanDragColumnsPipe} from './pipes/kanban-drag-columns.pipe';
 import {KanbanPostItDraggablePipe} from './pipes/kanban-post-it-draggable.pipe';
-import {BsDropdownModule} from 'ngx-bootstrap';
-import {FilterWritableCollectionsPipe} from './pipes/filter-writable-collections.pipe';
+import {FilterWritableResourcesPipe} from './pipes/filter-writable-resources.pipe';
 import {PickerModule} from '../../../shared/picker/picker.module';
 import {KanbanSelectedItemPipe} from './pipes/kanban-selected-item.pipe';
+import {KanbanRestrictedSelectItemIdsPipe} from './pipes/kanban-restricted-select-item-ids.pipe';
+import {DropdownModule} from '../../../shared/dropdown/dropdown.module';
+import {ChooseLinkDocumentModalComponent} from './modal/choose-link-document/choose-link-document-modal.component';
+import {ModalModule as NgxModalModule} from 'ngx-bootstrap';
```

Comment:
We should avoid importing the whole `ngx-bootstrap` and should use something like this instead:
```typescript
import { ModalModule as NgxModalModule } from 'ngx-bootstrap/modal';
```